Index: nemiver-0.7.3/src/common/nmv-object.cc
===================================================================
--- nemiver-0.7.3.orig/src/common/nmv-object.cc	2010-03-04 18:31:26.349484063 +0100
+++ nemiver-0.7.3/src/common/nmv-object.cc	2010-03-04 18:34:16.273732730 +0100
@@ -59,7 +59,7 @@
     *m_priv = *a_object.m_priv;
 }
 
-Object::Object&
+Object&
 Object::operator= (Object const &a_object)
 {
     if (this == &a_object)
Index: nemiver-0.7.3/src/langs/nmv-cpp-ast.cc
===================================================================
--- nemiver-0.7.3.orig/src/langs/nmv-cpp-ast.cc	2010-03-04 18:37:20.733483290 +0100
+++ nemiver-0.7.3/src/langs/nmv-cpp-ast.cc	2010-03-04 18:37:27.744982512 +0100
@@ -64,7 +64,7 @@
     m_int_value = a_t.get_int_value ();
 }
 
-Token::Token&
+Token&
 Token::operator= (const Token &a_t)
 {
     m_kind = a_t.get_kind ();
